README.md

```diff
@@ -80,12 +80,13 @@ The build process has the following package requirements:
 * build-essential
 * binutils-mips-linux-gnu
 * python3
+* libpng
 
 Under Debian / Ubuntu (which we recommend using), you can install them with the following commands:
 
 ```bash
 sudo apt-get update
-sudo apt-get install git build-essential binutils-mips-linux-gnu python3
+sudo apt-get install git build-essential binutils-mips-linux-gnu python3 libpng-dev
```

Comment:
Docker shouldn't be affected since it is basically a Linux image.
I don't have a machine with macOS or Cygwin to be able to test how to add this dependency there. I'll try googling a bit.

Reply:
I believe Docker will need to install the package as well, so the Dockerfile probably needs to be updated. And we should probably ask people with macOS or Cygwin setups to help and/or test the update.

Note: This type of situation is the main reason I'm always worried when we add support for more platforms

Reply:
Speaking of the dockerfile, I just noted it still installs qemu and the qemu-irix stuff. 
Since we removed qemu stuff from the readme, should we remove that from the dockerfile too?

Reply:
Yeah I think it's probably fine to remove the qemu stuff from the Dockerfile